app/controllers/developer_controller.rb

```diff
@@ -0,0 +1,10 @@
+class DeveloperController < ApplicationController
+  def list
+    @form = SearchDeveloperForm.new(permitted_params)
+    @developers = @form.search
+  end
+
+  def permitted_params
```

Comment:
@thupv 
you should change `permitted_params` to `list_params` or something else